src/components/connection_handler/include/connection_handler/connection_handler_impl.h

```diff
@@ -174,20 +175,21 @@ class ConnectionHandlerImpl
   /**
    * \brief Callback function used by ProtocolHandler
    * when Mobile Application initiates start of new session.
-   * \param connection_handle Connection identifier within which session has to
-   * be started.
-   * \param session_id Identifier of the session to be started
+   * Result must be notified through NotifySessionStartedResult().
+   * \param connection_handle Connection identifier within which session
+   * has to be started.
+   * \param sessionId Identifier of the session to be start
    * \param service_type Type of service
+   * \param protocol_version Version of protocol
    * \param is_protected would be service protected
-   * \param hash_id pointer for session hash identifier
-   * \return uint32_t Id (number) of new session if successful, otherwise 0.
+   * \param params configuration parameters specified by mobile
    */
-  virtual uint32_t OnSessionStartedCallback(
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/9f0795170b9cc46c6c8565ba967da6ae3bdbc613.